### PR TITLE
replaced calls to session hash w/ helper methods

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

**Comment by AnnJohn** [comment]
cc @Lukeghenco @PeterBell

File: app/controllers/tweets_controller.rb [hunk 1]
```diff
@@ -21,25 +20,24 @@ class TweetsController < ApplicationController
     if params[:content] == ""
       redirect to "/tweets/new"
     else
-      user = User.find_by_id(session[:user_id])
-      @tweet = Tweet.create(:content => params[:content], :user_id => user.id)
+      @tweet = Tweet.create(:content => params[:content], :user_id => current_user.id)
```
Comment (Lukeghenco) [inline app/controllers/tweets_controller.rb]: Thanks Gabe,

But we should actually change this to be `current_user.tweets.create(content: params[:content])`

Comment (gj) [inline app/controllers/tweets_controller.rb]: fixed per your suggestion @Lukeghenco

**Comment by Lukeghenco** [comment]
@gj this looks great keep up the great work?

**Pull request merged**